solidserver/soliderver.go

```diff
@@ -161,6 +164,7 @@ KeepTrying:
 func (s *SOLIDserver) GetVersion(version string) diag.Diagnostics {
 
 	apiclient := gorequest.New()
+	apiclient.Proxy(s.ProxyURL)
```

Comment:
It worries me that I duplicated this logic between here and the `Request` method. I could implement a "helper" for creating the `apiclient` (with proxy set). But overall, it seems more reasonable for `GetVersion` to be implemented using `Request` (but that refactoring I think is out of scope for this PR).

Let me know what you think?